### ISSUE TITLE
Enhancement  - try again button when no internet

### ISSUE THREAD

**Issue by Richard38907** [Malopieds/InnerTune#239]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

No internet connection, try again page instead of pages keeping loading

A screen like that should be nice:

![324738276-ad3aab3d-5a4b-4614-9317-bf55f6f8c574](https://github.com/user-attachments/assets/71ededdd-5d91-448e-8a27-1895bbddafb2)

### Why do you want this feature?

Easier to use and more friendly

**Comment by Malopieds** [comment]
Yes we could add a timer after which if no response a button appears. I don't know how doable it is as if now but I'll look into it

**Comment by Richard38907** [comment]
Thank you sir🫡
